Code.gs

```diff
@@ -174,6 +174,11 @@ function startSync(){
 
     //------------------------ Fetch URL items ------------------------
     var responses = fetchSourceCalendars(sourceCalendarURLs);
+      //Skip the source calendar if a 5xx or 4xx error is returned.  This prevents deleting all of the existing entries if the URL call fails.
+      if (responses.length == 0){
+        Logger.log("Error Syncing " + sourceCalendarName + ". Skipping...");
```

Comment:
I'm uncomfortable with this solution. Instead of a conspicuously missing calendar, we end up with a wrong calendar that appears to be correct (because it is merely outdated).

See discussion in https://github.com/derekantrican/GAS-ICS-Sync/issues/343#issuecomment-1858443442

Reply:
Compare with this approach: https://github.com/derekantrican/GAS-ICS-Sync/pull/403